logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java

```diff
@@ -76,6 +88,13 @@ public LoggingFraction applyDefaults(Level level) {
         return this;
     }
 
+    public static LoggingFraction getInstance() {
```

Comment:
To have one singleton / class of the LoggingFraction. Otherwise, we can't chain the methods.